example_configs/librenms.md

```diff
@@ -0,0 +1,195 @@
+# Configuration for LibreNMS
+
+You can either configure LibreNMS from the webui or from the command line. This is a list of the variables that you should set.
+
+## Essential
+
+## auth_ldap_uid_attribute
+
+```    
+uid
+```
+default: uidNumber
```

Comment:
Including the default is a bit confusing, it looks like it's a recommendation from LLDAP. I would just remove them to clear up any ambiguity

Reply:
Done.